### PR TITLE
Create Names.txt

### PR DESCRIPTION
**Pull request by prophet1906**

### CODE REVIEW

**Review by akki-akshat — DISMISSED** [review]

**Review by akki-akshat — COMMENTED** [review]
# Please add my name too

File: Names.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Shubham
```
Comment (prophet1906) [inline Names.txt:1]:
```suggestion
Shubham
Akshat
```

File: Names.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+Akshat
```
Comment (prophet1906) [inline Names.txt:2]:
```suggestion
Akshat123
```

File: Names.txt (hunk 1 above)
Comment (akki-akshat) [reply Names.txt:1]: ### That's awesome, thanks

**Pull request merged**